packages/perspective-viewer-d3fc/src/js/series/sunburstSeries.js

```diff
@@ -0,0 +1,137 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2017, the Perspective Authors.
+ *
+ * This file is part of the Perspective library, distributed under the terms of
+ * the Apache License 2.0.  The full license can be found in the LICENSE file.
+ *
+ */
+
+import {select, arc, interpolate} from "d3";
+
+export function sunburstSeries(sunburstElement, settings, split, data, color, radius) {
```

Comment:
I like this better. If we want to make it more d3-component-like, we could make it a function that returns a render-function, like so:

```javascript
export function sunburstSeries(settings, split, data, color, radius) {
  return sunburstElement => {
    // ...
  }
}
```

Reply:
I've made it more component-like and split into separate files. Would you want to give it another review?